src/Swift.Bindings/src/Emitter/StringEmitter/Handler/MethodHandler.cs

```diff
@@ -1007,13 +1045,6 @@ private void EmitFinally(CSharpWriter csWriter)
         {
             csWriter.WriteLine("finally");
```

Comment:
I guess we do not need the whole "finally block" then if all the allocations are done on stack.